### PR TITLE
Add BHNT 29 - closes #31

### LINKED ISSUE #31: Add list of hacks for BHNT#29
Sources:

* https://www.meetup.com/Berlin-Hack-and-Tell/messages/boards/thread/43828862
* https://www.meetup.com/Berlin-Hack-and-Tell/events/178757202/

### PR DESCRIPTION
**Pull request by ligi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -159,6 +159,15 @@ This event was mislabeled \#46 on Meetup.
...
 2014-04-29 | 18:30 | [Meetup Event](https://www.meetup.com/Berlin-Hack-and-Tell/events/178757202/) | [Photos](https://www.meetup.com/Berlin-Hack-and-Tell/photos/21582622/) | [@c-base](https://www.c-base.org)
 
+* [QT openglscroller](https://bitbucket.org/ni_cc/qtdeclarative/src/OpenGLScroller/examples/quick/scenegraph/openglunderqml) by nicc
+* [Nuclear Chess](https://github.com/karlb/nuclearchess) by [Karl](https://twitter.com/karlb) & Ulf
+* ATOM driven website by Volker
+* RoboCoding by Silver
+* [tumblr code hack/p5tweets](https://github.com/hamoid/Fun-Programming/blob/master/processing/ideas/2013/08/p5tweets/p5tweets.pde) by [Abe](http://funprogramming.org)
+* Cherry by [Santiago](https://twitter.com/cherryqueue)
+* Panoramico by Fernando
+* Mate Light Raytracer by Michael - **Hack of the month**
```
Comment (ligi) [inline README.md]: another place where it would be nice if @michaelkreil could know the link

Comment (MichaelKreil) [inline README.md]: It's becoming a full time job! ;)
How can I make changes? Or should I just post the line here?
* [Mate Light Raytracer](https://github.com/MichaelKreil/matelight-playground/blob/master/raytrace1.js) by [Michael Kreil](https://github.com/MichaelKreil) - **Hack of the month**

Comment (vog) [inline README.md]: You can either clone this project, make the changes to README.md, and create a pull request - the same way as with any other GitHub project.

Or you can click on "README.md", then "Edit", make the changes through the GitHub web interface, and save it is pull request.

Comment (vog) [inline README.md]: Created pull request #86.

**Pull request merged**